{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:56:32.279Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:56:32.283Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:56:32.284Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:58:01.407Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:58:01.412Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: failed to connect to server [127.0.0.1:27017] on first connect\n    at Pool.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/topologies/server.js:326:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:191:7)\n    at Connection.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/pool.js:270:12)\n    at Object.onceWrapper (events.js:293:19)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:194:7)\n    at Socket.<anonymous> (/Users/medeaohm/Desktop/Github/iOS-App-ChallengeAccepted/ChallengeAcceptedServer/node_modules/mongodb-core/lib/connection/connection.js:175:49)\n    at Object.onceWrapper (events.js:293:19)\n    at emitOne (events.js:96:13)","level":"warn","timestamp":"2017-04-01T07:58:01.414Z"}
{"code":209,"message":"invalid session token","level":"error","timestamp":"2017-04-01T09:23:58.708Z"}
{"code":209,"message":"invalid session token","level":"error","timestamp":"2017-04-01T09:24:33.864Z"}
